docs/rules/recommended.md

```diff
@@ -27,6 +27,12 @@ Errors:
 - [security-defined](./security-defined.md)
 - [spec-components-invalid-map-name](./spec-components-invalid-map-name.md)
 - [spec](./spec.md)
+- [parameters-unique](./arazzo/parameters-unique.md)
```

Comment:
It seems to me that we need different structure for rules documentation.
Current version does not diverse OAS rules from AsyncAPI and Arazzo rules.
Its easy to be confused and mix spec specific rules, without checking rule details.

cc: @lornajane , @tatomyr

Reply:
I agree. Maybe groups like "OpenAPI", and  "Arazzo".

Reply:
Or separate labels.

Reply:
Yes, definitely. I'll push some changes to this branch and try to make it easier to understand.

Reply:
Mmmm, I added the ruleset-templates page with all the rulesets for all the formats listed. Maybe I should have split that information onto these existing minimal and recommended documentation pages though, does anyone have an opinion?

Reply:
Updated to reflect only having documentation for per-API configuration and keeping the per-format information as an internal-only implementation detail for now.